docs/source/conf.py

```diff
@@ -157,44 +154,22 @@
 #
 #
 
-html_theme = 'sphinx_rtd_theme'
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+html_theme = 'pytorch_sphinx_theme'
+html_theme_path = [pytorch_sphinx_theme.get_html_theme_path()]
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
 html_theme_options = {
-    'collapse_navigation': False,
-    'display_version': True,
-}
-
-html_context = {
-    'css_files': [
-        'https://fonts.googleapis.com/css?family=Lato',
-        '_static/css/pytorch_theme.css'
-    ],
+    "collapse_navigation": False,
+    "display_version": True,
+    "logo_only": True,
+    "pytorch_project": "docs",
+    "navigation_with_keys": True,
+    "analytics_id": "UA-117752657-2",
```

Comment:
Is this PyTorch's google analytics ID?

Reply:
Yes, it is the same as in vision: 

https://github.com/pytorch/vision/blob/dad6e6a44d63272459b54175c92ed22955eb4207/docs/source/conf.py#L126

and pytorch:

https://github.com/pytorch/pytorch/blob/b9d4c0c7a9689982feb949f9609cec143c4a7268/docs/source/conf.py#L393

Reply:
Awesome!